### PR TITLE
Mini Project Inheritance - Polymorphism

### PR DESCRIPTION
**Pull request by Williiam8**

### CODE REVIEW

File: WilliamsGarcia/26-08/InheritanceExample/Main.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        MEMBERS_CLUB[1] = new Player("Victor", "Valdés", 25, 1, "GoalKeeper");
+        MEMBERS_CLUB[2] = new Player("Gianluca", "Zambrotta", 30, 11, "Defender");
+        MEMBERS_CLUB[3] = new Player("Eric", "Abidal", 28, 22, "Defender");
+        MEMBERS_CLUB[4] = new Player("Carles", "Puyol", 29, 5, "Defender");
+        MEMBERS_CLUB[5] = new Player("Gabriel", "Milito", 27, 3, "Defender");
+        MEMBERS_CLUB[6] = new Player("Xavi", "Hernández", 27, 6, "Midfielder");
+        MEMBERS_CLUB[7] = new Player("Andres", "Iniesta", 23, 8, "Midfielder");
+        MEMBERS_CLUB[8] = new Player("Yaya", "Touré", 24, 24, "Midfielder");
+        MEMBERS_CLUB[9] = new Player("Ronaldo", "De Assis", 27, 10, "Attacker");
+        MEMBERS_CLUB[10] = new Player("Leonel", "Messi", 20, 19, "Attacker");
```
Comment (edbermeo) [inline WilliamsGarcia/26-08/InheritanceExample/Main.java:29]: 30 is the new one :P

**Review by edbermeo — APPROVED** [review]
Great code. LGTM!

**Review by luisod78 — APPROVED** [review]
Nice job !

File: WilliamsGarcia/26-08/InheritanceExample/Main.java (hunk 1 above)
Comment (Williiam8) [reply WilliamsGarcia/26-08/InheritanceExample/Main.java:29]: Yes, I was based on the previous one. I feel old :'v

**Pull request merged**